policy/modules/system/userdomain.if

```diff
@@ -242,6 +243,34 @@ template(`userdom_user_content_access_template',`
 	')
 ')
 
+#######################################
+## <summary>
+##	Associate the specified domain to be
+##	a domain capable of executing other
+##	applications on behalf of the specified
+##	user.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <param name="userdomain_prefix">
+##	<summary>
+##	The prefix of the user domain (e.g., user
+##	is the prefix for user_t).
+##	</summary>
+## </param>
+## <rolebase/>
+#
+interface(`userdom_spec_user_application_exec_domain',`
```

Comment:
I'm not fond of the name of this, but don't immediately have a better name. Also, this is a template.

Reply:
`spec` is supposed to be for cases where setexec() is used to initiate a domain transition.

Reply:
how about `userdom_application_exec_domain`